### ISSUE TITLE
GCloud SDK version 404.0.0 can't be used

### ISSUE THREAD

**Issue by montss** [GoogleCloudPlatform/app-maven-plugin#480]
it seems that the appengine 2.0.8 which is based on the open source model appengine has very long paths which create those errors on Windows

plugin version: 2.4.4

maven config:

```
<plugin>
	<groupId>com.google.cloud.tools</groupId>
	<artifactId>appengine-maven-plugin</artifactId>
	<version>2.4.4</version>
	<configuration>
		<cloudSdkVersion>404.0.0</cloudSdkVersion>
	</configuration>
</plugin>
```

Error:
```
[INFO] --- appengine-maven-plugin:2.4.4:run (default-cli) @ ******* ---


Your current Google Cloud CLI version is: 404.0.0
Installing components from version: 404.0.0

+---------------------------------------------------+
|        These components will be installed.        |
+------------------------------+---------+----------+
|             Name             | Version |   Size   |
+------------------------------+---------+----------+
| Cloud Datastore Emulator     |   2.2.2 | 35.0 MiB |
| gRPC Python library          |  1.20.0 |  1.5 MiB |
| gcloud app Java Extensions   |   2.0.8 | 73.2 MiB |
| gcloud app Python Extensions | 1.9.101 |  8.6 MiB |
+------------------------------+---------+----------+

For the latest full release notes, please visit:
  https://cloud.google.com/sdk/release_notes

#============================================================#
#= Creating update staging area                             =#
#============================================================#
ERROR: gcloud crashed (FileNotFoundError): [WinError 3] The system cannot find the path specified: 'C:\\Users\\*******\\AppData\\Local\\Google\\ct4j-cloud-sdk\\404.0.0\\google-cloud-sdk.staging\\platform\\google_appengine\\google\\appengine\\tools\\java\\jetty94\\jetty-home\\demo-base\\webapps\\doc\\9.4.48.v20220622\\quick-start\\configuring\\images\\Jetty_Configuration_File_Relationships.png'

If you would like to report this issue, please run the following command:
  gcloud feedback

To check gcloud for common problems, please run the following command:
  gcloud info --run-diagnostics
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  22.683 s
[INFO] Finished at: 2022-09-28T14:39:48+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.google.cloud.tools:appengine-maven-plugin:2.4.4:run (default-cli) on project *******: Execution default-cli of goal com.google.cloud.tools:appengine-maven-plugin:2.4.4:run failed: com.google.cloud.tools.managedcloudsdk.command.CommandExitException: Process failed with exit code: 1 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Exception in thread "Thread-2" java.lang.NoClassDefFoundError: com/google/common/io/RecursiveDeleteOption
	at com.google.cloud.tools.managedcloudsdk.components.WindowsBundledPythonCopier.deleteCopiedPython(WindowsBundledPythonCopier.java:82)
	at com.google.cloud.tools.managedcloudsdk.components.WindowsBundledPythonCopier.lambda$copyPython$0(WindowsBundledPythonCopier.java:58)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ClassNotFoundException: com.google.common.io.RecursiveDeleteOption
	at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
	... 3 more
```

**Comment by mpeddada1** [comment]
Hi @montss thanks for filing the issue. To gain more information, did you notice this happening with older versions of the gcloud CLI?

**Comment by ludoch** [comment]
Seems on Windows only trying to delete an old path in the previous SDK...

Best workaround might be to delete the old cache area by hand and not by the Cloud SDK program itself..
Try to delete C:\\Users\\*******\\AppData\\Local\\Google\\ct4j-cloud-sdk\\. first and retry?

**Comment by ludoch** [comment]
Will be fixed when https://github.com/GoogleCloudPlatform/appengine-java-standard/commit/8b437c2fb694dfbf2ec72dd559cd1596dac0a918 is integrated in the Cloud SDK next version. For now, on Windows, stick with version 4.0.3

**Comment by montss** [comment]
> Hi @montss thanks for filing the issue. To gain more information, did you notice this happening with older versions of the gcloud CLI?

No it started with 4.0.4

@ludoch Thanks for the fix!

**Comment by burkedavison** [comment]
Fix applied in 2.0.9 release.
